### PR TITLE
Update kubernetes-api.md for zh

### LINKED ISSUE #18809: Delete extensions/v1beta1 in docs
**This is a Feature Request**

Since v1.16, extensions/v1beta1 has been deleted and it should not be used in docs

**What would you like to be added**
The following pages should be edited due to this deletion.
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/overview/kubernetes-api.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/workloads/controllers/deployment.md
- [x] https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/workloads/controllers/replicaset.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/reference/access-authn-authz/admission-controllers.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/reference/kubectl/conventions.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/reference/using-api/api-overview.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/access-application-cluster/ingress-minikube.md
- [x] https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md
- [x] https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/manage-kubernetes-objects/imperative-config.md

**Why is this needed**
extensions/v1beta1 is deleted in 1.16

**Comments**

### PR DESCRIPTION
**Pull request by huchengze**
This PR is going to fix #18809 .
update [kubernetes-api.md](https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/overview/kubernetes-api.md) for zh.
/language zh

### CODE REVIEW

File: content/zh/docs/concepts/overview/kubernetes-api.md [hunk 1]
```diff
@@ -244,22 +241,35 @@ to pick up the `--runtime-config` changes.
...
 该标志接受描述apiserver的运行时配置的逗号分隔的一组键值对。
 
-重要：启用或禁用组或资源需要重新启动apiserver和控制器管理器来使得 `--runtime-config` 更改生效。
+{{< note >}}
 
 <!--
-## Enabling resources in the groups
+Enabling or disabling groups or resources requires restarting apiserver and controller-manager
+to pick up the `--runtime-config` changes.
+-->
```
Comment (tengqm) [inline content/zh/docs/concepts/overview/kubernetes-api.md:249]: 不要将注释内容放在 note 内，可能出错。

**Comment by huchengze** [comment]
@tengqm 
并没有把注释内容放在 note 内，可以在这里看 https://github.com/huchengze/website/edit/patch-6/content/zh/docs/concepts/overview/kubernetes-api.md.

File: content/zh/docs/concepts/overview/kubernetes-api.md [hunk 2]
```diff
@@ -244,22 +241,36 @@ to pick up the `--runtime-config` changes.
...
+## 启用 extensions/v1beta1 组中资源
+
+在 `extensions/v1beta1` API 组中，DaemonSets，Deployments，StatefulSet, NetworkPolicies, PodSecurityPolicies 和 ReplicaSets 是默认禁用的。
+例如：要启用 deployments 和 daemonsets，请设置 `--runtime-config=extensions/v1beta1/deployments=true,extensions/v1beta1/daemonsets=true`。
+
+{{< note >}}
+
+<!--
+Individual resource enablement/disablement is only supported in the `extensions/v1beta1` API group for legacy reasons.
+-->
```
Comment (tengqm) [inline content/zh/docs/concepts/overview/kubernetes-api.md:271]: same here, move line 269-271 out of note shortcode.

**Comment by tengqm** [comment]
@huchengze still working on this?

**Comment by huchengze** [comment]
@tengqm Is this OK?
Sorry I haven't checked github these days.

**Pull request merged**